core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -1404,6 +1404,9 @@ class WorkspaceServiceSpec extends AnyFlatSpec with ScalatestRouteTest with Matc
     // the arguments passed to it so that we can verify that they were correct
     verify(services.servicePerimeterService).overwriteGoogleProjectsInPerimeter(servicePerimeterNameCaptor.capture)
     servicePerimeterNameCaptor.getValue shouldBe servicePerimeterName
+
+    // verify that we set the folder for the perimeter
+    verify(services.gcsDAO).addProjectToFolder(ArgumentMatchers.eq(workspace.googleProjectId), any[String])
```

Comment:
Can you add this verification to the cloneWorkspace version of this test as well? Around line 1545